packages/next/src/client/components/react-dev-overlay/internal/helpers/enqueue-client-error.ts

```diff
@@ -0,0 +1,22 @@
+import { isHydrationError } from '../../../is-hydration-error'
+
+// Dedupe the two consecutive errors: If the previous one is same as current one, ignore the current one.
+export function enqueueConsecutiveDedupedError(
+  queue: Array<Error>,
+  error: Error
+) {
+  const isFront = isHydrationError(error)
+  const previousError = isFront ? queue[0] : queue[queue.length - 1]
+  // Only check message to see if it's the same error, as message is representative display in the console.
+  if (previousError && previousError.message === error.message) {
+    return
+  }
```

Comment:
This is a good enought heuristic for now. But it deviates from how the browser does it. In the browser, errors are deduped by stack. That wouldn't dedupe errors from the second StrictMode render but that's not the goal. At least not until we checked why we're getting errors from the second StrictMode render in the first place when React DevTools is attached and console should be hidden during second render.

We're also missing a badge of how many errors there are. 

These can be fixed after 15.0 but as a fast follow-up